web/src/routes/Events.jsx

```diff
@@ -365,7 +379,7 @@ export default function Events({ path, ...props }) {
               download
             />
           )}
-          {(downloadEvent.end_time && downloadEvent.has_snapshot && !downloadEvent.plus_id) && (
+          {downloadEvent.end_time && downloadEvent.has_snapshot && !downloadEvent.plus_id && (
```

Comment:
looks like our vscode settings are fighting

Reply:
is there a specific setting or place you have yours setup from? I think I am just using whatever vscode decided, if there's a template hopefully I can import that